apps/astarte_realm_management/lib/astarte_realm_management/migrations/create_datastream_individual_multi_interface.ex

```diff
@@ -27,8 +27,8 @@ defmodule Astarte.RealmManagement.Migrations.CreateDatastreamIndividualMultiInte
       add :interface_id, :uuid, primary_key: true
       add :endpoint_id, :uuid, primary_key: true
       add :path, :varchar, primary_key: true
-      add :value_timestamp, :timestamp, partition_key: true
-      add :reception_timestamp, :timestamp, partition_key: true
+      add :value_timestamp, :utc_datetime_usec, partition_key: true
+      add :reception_timestamp, :utc_datetime_usec, partition_key: true
```

Comment:
As agreed upon IRL, let's move this to another commit

Reply:
This is no longer necessary as @noaccOS pointed out in [the comment below](https://github.com/astarte-platform/astarte/pull/1228#discussion_r2055677549)